scapy/main.py

```diff
@@ -16,6 +16,7 @@
 import logging
 from random import choice
 import types
+from io import open
```

Comment:
Sorry, can you not overwrite the built-in `open`? Using  `import io` and `io.open`, for example?

Reply:
Sorry for this silly change ... I updated the commit.